drv/fpga-api/src/lib.rs

```diff
@@ -118,14 +118,28 @@ pub enum BitstreamType {
     Compressed = 1,
 }
 
+/*
+    The Ops defined by the FPGA peripheral are:
+    Write = 0
+    Read = 1
+    BitSet = 2
+    BitClear = 3
+    NoOp = 4
+    WriteNoAddrIncr = 5
+    ReadNoAddrIncr = 6
+
+    The {Write,Read}NoAddrIncr operations support sequential write/read to the
+    same address. These operations are useful for exposing FIFO-like interfaces
+    via a single memory-mapped address location.
+*/
+
```

Comment:
I feel like the removed comment a couple lines down, explaining why the enum only contains a subset of the operations, was kinda useful. Adding a single line comment to each op enum saying something like

`/// This is just reads; for writes, see WriteOp`

would help avoid confusing readers, I think.